packages/docs/docs/concepts/progressive-loading/volumeProgressive.md

```diff
@@ -0,0 +1,134 @@
+The volume progressive loading extends the basic stack loading with the ability
+to interleave various images, interpolating them from a reduced resolution
+version both intra and inter image. That is, individual images might be fetched
+initially at 1/4 size (256x256 for a CT), and then only the initially displayed
+image plus every 4th image, with other images being interpolated. In this case,
+replicate interpolation is used to minimize interpolation overhead. Finally,
+after the lossy initial versions are fetched, the remaining images are fetched.
+
+The default retrieve ordering is below, where Decimate is described as the
+interval between images included, and the offset in that set.
+
+- Initial images, full resolution
+- Decimate 4/3 partial resolution
+  - Interpolate images -2...+1 (nearest neighbors)
+- Decimate 4/1 partial resolution
+- Decimate 4/2 full resolution
+- Decimate 4/4 full resolution
+- Decimate 4/3 full resolution
+- Decimate 4/1 full resolution
+
+The same ordering is done if partial resolution is not configured, except that
+the last two stages are never run because the partial resolution has already
+loaded those. This DOES allow the interpolation of results to appear very quickly.
+
+# Types of Partial Resolution
+
+There are a few types of partial resolution image:
+
+- lossy images are original resolution/bit depth, but lossy encoded
+- thumbnail images are reduced resolution images
+- byte range images are a prefix of the full resolution, followed by
+  retrieving the remaining data. This only works for images like HTJ2K encoded
+  in resolution first ordering.
+
+# Creating Partial Resolution Images
+
+Static DICOMweb has been enhanced to add the ability to create partial resolution
+images, as well as to serve up byte range requests. Some example commands
+for a Ct dataset are below:
+
+```bash
+# Create HTJ2K as default and write HTJ2K lossy to .../lossy/
+mkdicomweb create -t jhc --recompress true --alternate jhc --alternate-name lossy d:\src\viewer-testdata\dcm\Juno
+# Create JLS and JLS thumbnail versions
+mkdicomweb create -t jhc --recompress true --alternate jls --alternate-name jls /src/viewer-testdata/dcm/Juno
+mkdicomweb create -t jhc --recompress true --alternate jls --alternate-name jlsThumbnail --alternate-thumbnail /src/viewer-testdata/dcm/Juno
+# Create HTJ2K lossless and thumbnail versions (this is not required in general
+# when the top item is already lossless)
+mkdicomweb create -t jhc --recompress true --alternate jhcLossless --alternate-name htj2k  /src/viewer-testdata/dcm/Juno
+mkdicomweb create -t jhc --recompress true --alternate jhc --alternate-name htj2kThumbnail --alternate-thumbnail /src/viewer-testdata/dcm/Juno
+```
+
+Any other tools creating multipart/related encapsulated data can be used, as
+can using accept headers or parameters for a standard DICOMweb server.
+
+Note the data path for these is, in general the normal DICOMweb path with
+`/frames/` replaced by some other name.
+
```

Comment:
I still believe there should be no mention of the static DICOM web, and all the bits and pieces of the static DICOM web should have their own page, as it is not commonly used by the community. Our users often connect to open-source or commercial PACS, and they don't maintain that.